x-pack/plugins/observability_solution/inventory/common/entities.ts

```diff
@@ -57,3 +64,29 @@ export const entityTypesRt = new t.Type<EntityType[], string, unknown>(
   },
   (arr) => arr.join()
 );
+
+interface BaseEntity {
+  [ENTITY_LAST_SEEN]: string;
+  [ENTITY_ID]: string;
+  [ENTITY_TYPE]: EntityType;
+  [ENTITY_DISPLAY_NAME]: string;
+  [ENTITY_DEFINITION_ID]: string;
+}
+
+interface ServiceEntity extends BaseEntity {
+  [ENTITY_TYPE]: 'service';
+  'service.name': string;
```

Comment:
You can import it from: https://github.com/elastic/kibana/blob/49661a35744adea396d6a80039d1a5f9976c957e/x-pack/plugins/observability_solution/observability_shared/common/field_names/elasticsearch.ts#L15